sympy/solvers/solveset.py

```diff
@@ -9,8 +9,8 @@
 from sympy.core.function import (Lambda, expand, expand_complex)
 from sympy.core.relational import Eq
 from sympy.simplify.simplify import fraction, trigsimp
-from sympy.functions import (log, Abs, tan, cot, exp,
-                             arg, Piecewise, piecewise_fold)
+from sympy.functions import (log, Abs, sin, cos, sec, csc, tan, cot,
+                            exp, arg, Piecewise, piecewise_fold)
```

Comment:
Keep the imports in alphabetical order -  as import lists grow, any other order will become unmanageable.